english/wikipedia/wiki.md

```diff
@@ -31,10 +31,14 @@ but seemingly everyone makes use of Wikipedia!
 
 Until you get bored and go explore [here](https://www.urbandictionary.com) instead.
 
-Then after you explore the website, you learn a few neat tricks. 
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.